### ISSUE TITLE
Regression: Image custom object is not properly handled in RichText demo

### ISSUE THREAD

**Issue by afester** [TomasMikula/RichTextFX#584]
* Launch the RichTextDemo
* Insert an image
* Place cursor to start of paragraph
* Press Enter
=> Image is shown twice, once in the upper (new) paragraph and once in the lower paragraph.
When using an older variant of the LinkedImageOps, it works well.

**Comment by afester** [comment]
Modifying realSubSequence in NodeSegmentOpsBase so that it creates an empty segment when the sub sequence length is 0 fixes it:

```
    @Override
    public SEG realSubSequence(SEG seg, int start, int end) {
        if (start == end) {
            return createEmpty();
        }
        return seg;
    }
```
Needs some additional testing before submitting a PR, but I think this should be a safe assumption in any case ...

**Comment by JordanMartinez** [comment]
This issue no longer arises in #590

**Comment by JordanMartinez** [comment]
Closed by #590
